app/Http/Controllers/Auth/GithubController.php

```diff
@@ -30,14 +31,16 @@ public function handleProviderCallback()
         try {
             $user = User::findByGithubId($socialiteUser->getId());
 
-            return $this->userFound($user);
+            return $this->userFound($user, $socialiteUser);
         } catch (ModelNotFoundException $exception) {
             return $this->userNotFound(new GithubUser($socialiteUser->user));
         }
     }
 
-    private function userFound(User $user): RedirectResponse
+    private function userFound(User $user, $socialiteUser): RedirectResponse
```

Comment:
Can you type-hint the socialite user here? It should be an instance of `Laravel\Socialite\Two\User`.